static/js/src/publisher-admin/pages/Collaboration/InviteCollaborator.tsx

```diff
@@ -36,28 +38,28 @@ function InviteCollaborator({ setShowSidePanel }: Props): JSX.Element {
   const setInviteLink = useSetRecoilState(inviteLinkState);
   const invitesList = useRecoilValue(invitesListState);
   const inviteLink = useRecoilValue(inviteLinkState);
+  const publisher = useRecoilValue(publisherState);
+  const collaboratorsList = useRecoilValue(filteredCollaboratorsListState);
 
   const [copied, setCopied] = useState(false);
   const [loadingInviteLink, setLoadingInviteLink] = useState(false);
 
-  const isUnique = (email: string | undefined) => {
-    if (!email) {
-      return true;
-    }
+  const hasPendingInvite = (email: string | undefined) =>
+    invitesList?.some(
+      (invite: Invite) => invite.email === email && isPending(invite)
+    );
 
-    if (!invitesList) {
-      return false;
-    }
+  const isCollaborator = (email: string | undefined) =>
+    collaboratorsList.some(
+      (collaborator) => collaborator?.account?.email === email
+    ) || publisher?.email === email;
 
-    const existingInvites = invitesList.filter((invite: Invite) => {
-      return invite.email === email && isPending(invite);
-    });
-
-    if (!existingInvites.length) {
+  const isValidEmail = (email: string | undefined) => {
+    if (!email) {
       return true;
     }
```

Comment:
This part of the function would return true if the email is undefined or an empty string. Is this what is intended? I expect that undefined or empty string should return false that invalid email.